routes/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "no-param-reassign": "off"
```

Comment:
Shouldnt this be moved to a different pr that deals with keywords? I have changed articles to use `findByIdAndUpdate` for now